serverless/experimental/gcp-log/gcs_uploader/function.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2021 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Package p provides a Google Cloud Function to write an object to GCS.
+package p
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	gcs "cloud.google.com/go/storage"
+)
+
+// CreateGCSObject is writes a GCS object with EntryContent to EntryPath in
```

Comment:
nit: "is writes"

also, should `EntryContent` etc. be `entryContent` to match the JSON naming?